annotation/src/main/java/org/cbioportal/genome_nexus/annotation/web/AnnotationController.java

```diff
@@ -225,6 +225,32 @@ public AnnotationController(VariantAnnotationService variantAnnotationService,
         return isoformOverrideService.getIsoformOverrides(source);
     }
 
+    @ApiOperation(value = "getIsoformOverrideSources",
```

Comment:
Minor, but maybe put a short sentence describing the endpoint like you do for the other ones.